Frameworks/UIKit/UIView.mm

```diff
@@ -2252,12 +2251,16 @@ - (CGPoint)convertPoint:(CGPoint)pos fromView:(UIView*)fromView {
 */
 - (void)drawLayer:(CALayer*)layer inContext:(CGContextRef)context {
     UIGraphicsPushContext(context);
+    _CGContextPushBeginDraw(context);
```

Comment:
Contexts that go onto the UIGraphics context stack cannot actually be safely assumed to only need their results after PopContext().

---
In reply to: [96087015](https://github.com/Microsoft/WinObjC/pull/1705#discussion_r96087015) [](ancestors = 96087015)